### DISCUSSION TITLE
Is it possible to make money with Crypto trading?

### DISCUSSION THREAD

**Discussion by ansarizafar** [bludnic/opentrader#60] [Q&A] [answered]
I am new to trading. Is it really possible to make money with opentrader and which strategy is profitable?

**Comment by bludnic** [comment, marked as answer]
OpenTrader doesn't provide bot configuration presets. You need to do your own research and configure the strategy params accordingly, which will determine profitability. I recommend using Paper Trading to test your ideas without risking real money.

### Supported strategies

- **GRID Bot Strategy:** The Grid strategy places buy and sell orders in a way that creates a "grid". It's designed to profit from market volatility by executing trades as the price moves within these intervals. Unlike most implementations, which only allow setting high/low prices, in OpenTrader you can configure each grid line's price and quantity, achieving an asymmetric grid. This enables placing orders at support and resistance levels or in regions with significant price fluctuations.

![image](https://github.com/user-attachments/assets/6e78ff4c-d9e9-4235-9b64-2112bdc85cbb)

- **RSI Strategy**: This strategy triggers a trade when the RSI indicator crosses `high` or `low` values (default to 70/30). You can also adjust the `persistence` param, which controls how many candles the RSI must stay within a trend before a trade is executed.

![image](https://github.com/user-attachments/assets/b3d901bc-6297-4df8-90af-84dd64238802)

### Upcoming strategies

- **DCA Strategy**: Dollar-Cost Averaging strategy that buys an asset at regular intervals to reduce the entry price of a position.
- **ARB Strategy**: Arbitrage strategy that takes advantage of price differences through cross-exchange trading.

Also, a Wiki on creating custom strategies (powered by auto-generated UI) is coming.

**Reply by ansarizafar** [reply]
Thanks for your reply. I am completely new to trading so I need guidance. I wanted to know, are you or anyone you know making money with opentrader? I would be grateful If you can help me learn and make few profitable trades so that I can gain the confidence and the basic understanding of the whole process.

**Reply by bludnic** [reply]
Remember, this is just a tool to automate your trading strategies. For guidance, you can try asking in the OpenTrader Community on Telegram, but keep in mind the community is still small. At the moment, I'm also collecting feedback from users to better understand the direction of the project. Feel free to share your ideas!

**Reply by ansarizafar** [reply]
Telegram is not available in my country. Why not use discord for community support?

**Reply by bludnic** [reply]
Added Discord invitation link in README.md
